### PR TITLE
disallow "publish", and "send  to" options in bulk actions menu for published items

### PR DESCRIPTION
**Pull request by tomaskikutis**
SDESK-5959

### CODE REVIEW

**Review by petrjasek — APPROVED** [review]

File: scripts/apps/search/controllers/get-bulk-actions.tsx [hunk 1]
```diff
@@ -26,7 +26,8 @@ export function getBulkActions(
 
     const {isLocked, isLockedByOtherUser, isPublished} = sdApi.article;
 
-    const noneLocked = articles.every((article) => !isLocked(article));
+    const noneLocked = articles.some((article) => isLocked(article)) !== true;
```
Comment (petrjasek) [inline scripts/apps/search/controllers/get-bulk-actions.tsx:29]: this is actually rather `someUnlocked` than `noneLocked`

Comment (tomaskikutis) [reply scripts/apps/search/controllers/get-bulk-actions.tsx:29]: `articles.some((article) => isLocked(article))` means `at least one is locked`.

If it is `not true`, that `at least one is locked`, then none are locked.

Comment (petrjasek) [reply scripts/apps/search/controllers/get-bulk-actions.tsx:29]: I've missed that `!== true` at the end, not very readable imo

**Review by petrjasek — APPROVED** [review]

File: scripts/apps/search/controllers/get-bulk-actions.tsx (hunk 1 above)
Comment (tomaskikutis) [reply scripts/apps/search/controllers/get-bulk-actions.tsx]: I thought `some` would perform better than `every`, but turns out it's the same. `every` stops checking after the first negative response.

**Pull request merged**